patches.json

```diff
@@ -95,11 +95,135 @@
   },
   "MC-35984": {
     "magento/magento2-ee-base": {
-      "Fixes issue with the broken Returns page after creating a shipping label for RMA": {
+      "Fixes the issue with the broken Returns page after creating a shipping label for RMA": {
         "2.4.0": {
           "file": "commerce/MC-35984__fixes_issue_in_rma_edit_page_with_shipping_label__2.4.0.patch"
         }
       }
     }
+  },
+  "MDVA-30195": {
+    "magento/magento2-base": {
+      "Fixes the issue where cron jobs fail if database name is too long, resulting in categories not being updated on the frontend.": {
+        ">=2.3.1 <=2.3.4-p2": {
+          "file": "os/MDVA-30195__fix_to_handle_long_database_names__2.3.4-p2.patch"
+        }
+      }
+    }
+  },
+  "MDVA-30106": {
+    "magento/magento2-base": {
+      "Fixes the issue where during checkout, payments are not loaded with “Cannot read property ‘length’ of null ” error in JS console.": {
+        ">=2.3.0 <=2.4.0": {
```

Comment:
Please, take into account that this fix wasn't delivered into mainline, so won't be available in Magento >2.4.0
If you believe this patch is important and should be applicable to all future Magento versions - use "^2.3.0" which is equivalent to ">=2.3.0 < 3.0.0"

Reply:
fixed